src/components/pages/HomePage.vue

```diff
@@ -1,10 +1,89 @@
+                    genomes</span> based on peptide similarity.
+                </v-col>
+                <v-col class="col-lg-6 col-sm-12">
+                    <v-card>
+                        <v-list-item two-line class="blue darken-1">
+                            <v-list-item-content>
+                                <v-list-item-title class="text-h5 white--text">
+                                    Unipept 4.3
+                                </v-list-item-title>
+                                <v-list-item-subtitle class="white--text">posted on 2020-04-15</v-list-item-subtitle>
+                            </v-list-item-content>
+                        </v-list-item>
+
+                        <v-card-text>
+                            Unipept 4.3 has just been released and contains these major additions:
+                            <ul>
+                                <li>
+                                    Load and process multiple samples at once with the metaproteomics analysis tool.
+                                </li>
+                                <li>
+                                    Compare multiple samples with each other using the new heatmap.
+                                </li>
+                                <li>
+                                    Functional analysis has been expanded with support for InterPro annotations.
+                                </li>
+                                <li>
+                                    UniProt updated to version 2020.01
+                                </li>
+                            </ul>
+                        </v-card-text>
+
+                        <v-divider class="mx-4"></v-divider>
+
+                        <v-card-actions>
+                            <v-spacer />
+                            <router-link
+                                to="/posts"
+                                v-slot="{ href, navigate }"
+                            >
+                                <v-btn
+                                    color="secondary"
+                                    text
+                                    @click="navigate"
+                                    :href="href"
+                                >
+                                    More news
+                                </v-btn>
+                            </router-link>
+                        </v-card-actions>
+                    </v-card>
+                </v-col>
+            </v-row>
+            <v-row>
+                <v-col v-for="(feature, index) in features" :key="index" class="col-lg-4 col-md-6">
+                    <router-link :to="feature.link">
+                        <v-hover>
+                            <template v-slot:default="{ hover }">
+                                <v-card :elevation="hover ? 6 : 2">
```

Comment:
You could extract these cards for the homepage into a separate Vue-component and work with slots in order to provide the body of the card. This way you don't need to pass the raw HTML content into the card using the `v-html` feature of Vue, but you could build something like this:

```
<feature-card >
    <template v-slot:body>
        <div>
        With tryptic peptide analysis, you can submit a single ...
        </div>
    </template>
</feature-card>
```

Reply:
Right now, if we change the width of the homepage such that only 2 cards are shown on one row, the height of the cards is not all equal. I think it would be better if we expand the smallest card in this case such that all cards on the same row are equally tall. See following screenshots:

Before:
<img width="714" alt="image" src="https://user-images.githubusercontent.com/9608686/185144401-73d3053c-207a-4031-bef8-bd7e183f0730.png">

After:
<img width="689" alt="image" src="https://user-images.githubusercontent.com/9608686/185144676-91a0aee1-0dcb-4717-aa74-5ca534c91e86.png">

Reply:
Both should be fixed with the last two commits.